spec/serializers/message_draft_serializer_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe MessageDraftSerializer, type: :serializer do
```

Comment:
Don't need this serializer I don't think -- just use the one for Message -- pass in the appropriate class. And in your specs you can pass in opts to the serializer helper to do the same.